pinecone/manage.py

```diff
@@ -194,14 +190,14 @@ def describe_index(name: str):
     """
     api_instance = _get_api_instance()
     response = api_instance.describe_index(name)
-    db = response['database']
-    ready = response['status']['ready']
-    state = response['status']['state']
-    return IndexDescription(name=db['name'], metric=db['metric'],
-                            replicas=db['replicas'], dimension=db['dimension'], shards=db['shards'],
-                            pods=db.get('pods', db['shards'] * db['replicas']), pod_type=db.get('pod_type', 'p1'),
```

Comment:
Do we want to expose `shards` and `replicas` if they are in the dict?  or should we handle it like it used to?

Reply:
good point, forgot about that. I will replace it with pods

Reply:
Also, we have been exposing shards and replicas always, just added pods for backward compatibility. We can continue doing that

Reply:
Oh right, missed that